pom.xml

```diff
@@ -68,6 +68,7 @@
 
     <modules>
         <module>api</module>
+        <module>formats/json-jackson</module>
```

Comment:
Another thing nice to see! #113

Reply:
Yep, now `cloudevents-api` depends on nothing!